triage/forms.py

```diff
@@ -126,11 +148,21 @@ def get_used_marketplace(answers):
     return answers.get('used_online_marketplace') is True
 
 
+def get_exporting_goods(answers):
```

Comment:
is this needed? Does not seem to be used anywhere

Reply:
that is true. i thought it'd be needed but isn't currently. will remove